MAINTAINERS.md

```diff
@@ -2,3 +2,4 @@
 
 * Dzmitry Shuiski <dzmitryshuiski@mlabs.city>
 * Marcus Fernandes <marcus@mlabs.city>
+* Szabo Gergely <gergely@mlabs.city>
```

Comment:
If we're sticking to the English name order, it would be Gergely Szabo

Reply:
My apologies. It should be fixed now